example/books/books/books.capnp.go

```diff
@@ -0,0 +1,103 @@
+// Code generated by capnpc-go. DO NOT EDIT.
```

Comment:
Since this isn't really intended for people to fetch with go-get, maybe we should just not commit this to the repo? I generally dislike having build artifacts stored in git, and it seems like we don't actually need it here. Same goes for the other example(s).

Also, you comitted book.capnpenc, which we *definitely* don't need to version.

Reply:
Yeah - good question. I did commit this generated file because in one OSS community I work in (Cosmos SDK) they have a requirement to commit the go files generated from their protocol buffers. Since the code that calls this generated file, depends on it, I could totally agree with that decision and leave it checked-in. Or... since it is generated from the .capnp, I could also remove it (which is what I would have done before I heard of projects that wanted the generated files committed). As maintainers, you might have to make a decision... I don't mind either way. I will remove book.capnpenc, although that is committed for the same reason... so I'll maybe also remove the generated go files too. Sound OK?